src/extensions/defaultExtension.ts

```diff
@@ -0,0 +1,5 @@
+import { ITemplateExtension } from './ITemplateExtension';
```

Comment:
Please delete this file.
If there are no default extensions this file can be left out :)

Reply:
Agreed